travis/install_custom_pg

```diff
@@ -15,8 +15,14 @@ pkgingurl='https://anonscm.debian.org/git/pkg-postgresql/postgresql.git'
 patchurl="${pkgingurl}/plain/debian/patches/50-per-version-dirs.patch"
 
 # clone PostgreSQL
+if [ "${PGVERSION}" -eq '10' ]; then
+    gitref='REL_10_BETA1'
+else
+    gitref="REL${PGVERSION//./_}_STABLE"
+fi
+
```

Comment:
I'm not quite sure what the point of using BETA1 rather than master is.  That just seems to hide upcoming breakage?